tests/frame_test.go

```diff
@@ -72,3 +72,36 @@ func TestFrameDismissDialogBox(t *testing.T) {
 		})
 	}
 }
+
+func TestFrameNoPanicWithEmbeddedIFrame(t *testing.T) {
+	// This test only works when headless mode is false.
+	t.SkipNow()
+
+	t.Parallel()
+
+	b := newTestBrowser(t, withFileServer())
+	p := b.NewPage(nil)
+
+	var result string
+	err := b.await(func() error {
+		opts := b.toGojaValue(struct {
+			WaitUntil string `js:"waitUntil"`
+		}{
+			WaitUntil: "networkidle",
+		})
+		pageGoto := p.Goto(
+			b.staticURL("embedded_iframe.html"),
```

Comment:
Did you forget to commit this file? The test fails with a 404 for me, even when enabling headed mode.

Reply:
Thanks for reminding me. Added in 89221d54dfd64d84a879b368eb1d344fa83e244c